packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -90,6 +90,24 @@ function Root({
       []
     ).length > 0
 
+  const menuItems = React.Children.toArray(children)
```

Comment:
Could we memoize this value like the other example above please? Alternatively, store the value of `React.Children.toArray(children)` in a const, and reference that in place of calls to `toArray` in multiple places. Conscious that we could be unnecessarily duplicating effort and impacting rendering performance otherwise.

Reply:
Great suggestion! I'll definitely utilize this hook more. I'm wondering, would it still be beneficial to store `React.Children.toArray(children)` in a const to replace usage of it directly? Or is the `useMemo` usage here enough?

Reply:
Sounds good, I'll leave it as is with the usage of `useMemo` then. Thanks too!